cmd/generate/config/rules/sentry.go

```diff
@@ -22,3 +24,63 @@ func SentryAccessToken() *config.Rule {
 	tps := utils.GenerateSampleSecrets("sentry", secrets.NewSecret(utils.Hex("64")))
 	return utils.Validate(r, tps, nil)
 }
+
+func SentryOrgToken() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "sentry-org-token",
+		Description: "Found a Sentry.io Organization Token, risking unauthorized access to error tracking services and sensitive application data.",
+		Regex:       regexp.MustCompile(`\bsntrys_eyJpYXQiO[a-zA-Z0-9+/]{10,200}(?:LCJyZWdpb25fdXJs|InJlZ2lvbl91cmwi|cmVnaW9uX3VybCI6)[a-zA-Z0-9+/]{10,200}={0,2}_[a-zA-Z0-9+/]{43}\b`),
+		// base64-encoded json fragments: eyJpYXQiO = `{"iat":`,
+		// depending on alignment: LCJyZWdpb25fdXJs = `,"region_url`, InJlZ2lvbl91cmwi = `"region_url"`, cmVnaW9uX3VybCI6 = `region_url":`
+
+		Entropy:  3.5,
```

Comment:
The entropy of these seem to be well above 5. I'd do at least 4.5.
```suggestion
		Entropy:  4.5,
```